src/systems/ec/implementations/vanilla/packages/ec-component-page-banner/ec-component-page-banner.scss

```
@@ -0,0 +1,239 @@
+/**
+ * Page Banner component
+ * @define page-banner
+ */
+
+// Import base
+@import '@ecl/ec-base/ec-base';
+
+// Check if overridden dependencies are already loaded, if needed
+@include check-imports(('ec-component-button'));
+
+@mixin ecl-page-banner(
+  $background-color-default: $ecl-color-grey-5,
+  $title-color-default: $ecl-color-black-100,
+  $baseline-color-default: $ecl-color-grey-100,
+  $background-color-primary: $ecl-color-primary,
+  $title-color-primary: $ecl-color-white-100,
+  $baseline-color-primary: $ecl-color-white-100,
+  $background-color-image: $ecl-color-white-100,
+  $title-color-image: $ecl-color-black-100,
+  $baseline-color-image: $ecl-color-grey-100,
+  $border-color-image: $ecl-color-yellow-100,
+  $background-color-image-shade: rgba(0, 0, 0, 0.7),
+  $title-color-image-shade: $ecl-color-white-100,
+  $baseline-color-image-shade: $ecl-color-white-100,
+  $content-width: 42.5rem
+) {
+  .ecl-page-banner {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin: 0;
+    position: relative;
+  }
+
+  .ecl-page-banner__container {
+    display: flex;
+    width: 100%;
+  }
+
+  .ecl-page-banner__content {
+    display: block;
+    padding: $ecl-spacing-l;
+  }
+
+  .ecl-page-banner__image {
+    background-size: cover;
+    height: (1/4) * 100vw;
```

yeah, or just `25vw` :)

```suggestion
    height: 25vw;
```

---

yes, it works too, but I find more explicit this way (we see the ratio directly)